<launch>
  <node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd_to_pointcloud" args="/home/goa/pcl_daten/table/icp_fov/pc_aligned_2.pcd 1" output="screen">
    <remap from="/cloud_pcd" to="/aggregate_point_map/pc_aligned" />
    <param name="frame_id" type="string" value="/map" />
  </node>
    <!--node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd_to_pointcloud2" args="/home/goa/pcl_daten/table/icp_fov/pc_aligned_2.pcd 1" output="screen">
    <remap from="/cloud_pcd" to="/pc_aligned_and_normals" />
    <param name="frame_id" type="string" value="/map" />
  </node-->
  </launch>